### PR TITLE
Adding support for amp_ad puffnetwork

### PR DESCRIPTION
**Pull request by yash-pn**
Adding support for amp_ad puffnetwork

```
<amp-ad width="300" height="250"
	type="puffnetwork"
	data-chid=233 >
</amp-ad>
```

### CODE REVIEW

**Comment by yash-pn** [comment]
Hi @zhouyx , can you please check my PR. This is for supporting ad-network 'puffnetwork'.

**Comment by yash-pn** [comment]
cc: @lannka  - Hi, please let me know if I am missing anything or this PR is it good to be merged?

**Comment by zhouyx** [comment]
Hi @yash-pn Sorry for the late response. I am currently out of office. To @ampproject/ads to review. @lannka please reassign reviewer. Thank you.

File: ads/_config.js [hunk 1]
```diff
@@ -807,6 +807,10 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  'puffnetwork': {
+    prefetch: 'https://static.puffnetwork.com/amp_ad.js',
```
Comment (lannka) [inline ads/_config.js]: please implement renderStart API: https://github.com/ampproject/amphtml/blob/master/ads/README.md#other-tips

Comment (yash-pn) [reply ads/_config.js]: I have implemented support renderStart API. Please review.

**Comment by yash-pn** [comment]
Hi @lannka I have implemented the changes requested. Please review.

**Pull request merged**